duckduckgo/iOS
# Sync: Send pixels for account removal + decoding issues

Task/Issue URL: https://app.asana.com/0/1201493110486074/1208723035104886/f
Tech Design URL:
CC:

**Description**:

Adds sync pixels for catching when the keychain read throws a decoding error and when the account is deleted for various reasons

**Steps to test this PR**:
1. Make sure you've already activated sync.
2. Deactivate sync (either delete the server data or just turn it off)
3. You should see the `sync_account_removed_reason_user-turned-off` pixel in the console.



**Definition of Done (Internal Only)**:


**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
